### PR TITLE
Use the new useglobal form attribute in captcha config options fields

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Improvement.

### Summary of Changes

This PR makes use of the new use global on the capatcha fields in com_users, com_contact and com_content captcha config options.

### Testing Instructions

1. Use latest staging and apply patch
2. Confirm with/without captacha plugins enabled the options still work and properly inherit the setting from global config.
3. Confirm capctha is, when enabled, still displayed in all frontend scenarios (create new article, submit contact form, users register, user remimd username, user password reset)

#### Before
image

#### After
image

### Documentation Changes Required

None.

### CODE REVIEW

**Comment by Quy** [comment]
Change `Use Default` to `Use Global`. 

useglobal

**Comment by andrepereiradasilva** [comment]
@Quy updated langauge strings

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9b12cdd6b1cfb4f0859fd1c9cbb1ce57026bc0d0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16573">issues.joomla.org/tracker/joomla-cms/16573</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 9b12cdd6b1cfb4f0859fd1c9cbb1ce57026bc0d0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16573">issues.joomla.org/tracker/joomla-cms/16573</a>.</sub>

File: libraries/joomla/form/fields/list.php [hunk 1]
```diff
@@ -166,11 +166,28 @@ protected function getOptions()
...
 			$value  = $params->get($this->fieldname);
+			$text   = '';
 
 			// Try with global configuration
 			if (is_null($value))
 			{
 				$value = JFactory::getConfig()->get($this->fieldname);
+
+				// For global config plugins field types (ex: editors) is also needed to load the language files.
+				if (strtolower($this->type) === 'plugins')
```
Comment (Bakual) [inline libraries/joomla/form/fields/list.php]: If it's only needed for the JFormFieldPlugins, we should better put that code into JFormFieldPlugins::getOptions instead. I don't like having fieldtype specific code in the library list field.

**Comment by andrepereiradasilva** [comment]
@Bakual chabged as requested please check and test

**Comment by Bakual** [comment]
Looks much better to me, thanks!

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e2afd383f6d29cb5878a219189cbcde36c13165b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16573">issues.joomla.org/tracker/joomla-cms/16573</a>.</sub>

**Pull request merged**